aggie-coding-club/Rev-Registration
# Delete schedule dialog

## Description

Creates a dialog when the user attempts to delete a saved schedule, also fixes the touch ripple of schedules playing whenever a schedule button is clicked

## Rationale

I used `useRef()` on the text component to determine the position of the save/delete buttons, since they have to have `position: absolute` to not trigger the touch ripple from the schedule list item.

## How to test
Save a schedule and try to delete it, the dialog will appear. You can use the buttons or press escape/enter to control it.

## Screenshots
![image](https://user-images.githubusercontent.com/28655462/91649356-bdb44e80-ea38-11ea-852c-7063c220bff1.png)


## Related tasks
We never made a task for this but I was supposed to work on it ¯\\_(ツ)_/¯
